### PR TITLE
Vector2: added angle() method

### PR DESCRIPTION
**Pull request by WestLangley**
Implements #7925.

### CODE REVIEW

File: src/math/Vector2.js [hunk 1]
```diff
@@ -370,6 +370,18 @@ THREE.Vector2.prototype = {
 
 	},
 
+	angle: function () {
+
+		// computes the angle in radians with respect to the positive x-axis
+
+		var angle = Math.atan2( this.y, this.x );
+
+		if ( angle < 0 ) angle += 2 * Math.PI;
```
Comment (mrdoob) [inline src/math/Vector2.js]: I wonder if we could pass a boolean tho the method for this?

Comment (WestLangley) [inline src/math/Vector2.js]: I do not understand your comment. `atan2` returns a value between `- PI` and `+ PI`. A more natural range is zero to `2 PI`. That is what this method returns.

Comment (mrdoob) [inline src/math/Vector2.js]: I see I see. It was in case someone expected the `- PI` to `+PI` output. Looks good then!

Comment (WestLangley) [inline src/math/Vector2.js]: If you're happy, I'm happy.  : - )

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**